README.md

```diff
@@ -46,6 +46,8 @@ dependencies {
 }
```

Comment:
I suggest adding strikethrough on line 35 and 45 to prevent people from adding it to their build.gradle:

* Maps SDK via Google Play Services ~~OR [Maps SDK v3 BETA] library~~

...but unfortunately GitHub doesn't seem to recognize strikethrough in a codeblock:

```groovy
    // Utilities for Maps SDK for Android (requires Google Play Services) 
    implementation 'com.google.maps.android:android-maps-utils:2.2.5'

    ~~// Alternately - Utilities for Maps SDK v3 BETA for Android (does not require Google Play Services)~~
    ~~implementation 'com.google.maps.android:android-maps-utils-v3:2.2.5'~~
```

So maybe add a deprecation notice here too?

```groovy
    // Utilities for Maps SDK for Android (requires Google Play Services) 
    implementation 'com.google.maps.android:android-maps-utils:2.2.5'

    // DEPRECATED - Alternately - Utilities for Maps SDK v3 BETA for Android (does not require Google Play Services)
    implementation 'com.google.maps.android:android-maps-utils-v3:2.2.5'
```

Reply:
Good suggestion! Will update